transport/src/androidTest/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImplTest.kt

```diff
@@ -1457,6 +1457,93 @@ class MessagingClientImplTest {
         }
     }
 
+    @Test
+    fun whenClearConversationFromAnyNonReadOnlyOrConfiguredState() {
```

Comment:
What about testing the `clearConversation` activation when the session is an authenticated session?
According to the feature charter, clearing an authenticated session should not be enabled.
> see Authenticated Use-cases on
 https://intranet.genesys.com/display/PCINIT/PURE-4519+-+Messenger+-+End-User+can+Clear+Guest+Session

Reply:
Took me a while, but you are looking at the wrong charter. Here is the right one:
https://intranet.genesys.com/pages/viewpage.action?pageId=277722827